data/hai ships.txt

```diff
@@ -87,7 +87,7 @@ ship "Flea"
 	engine 0 27
 	gun 0 -27 "Railgun"
 	explode "tiny explosion" 15
-	description `This is a combat drone designed to help contain Unfettered aggression with a minimum risk to Hai life.`
+	description `The Flea is a light combat ship designed to help contain Unfettered aggression, while keeping a minimum risk to Hai life using remote control rather than pilots.`
```

Comment:
I took the remote control from the Pond Strider's description, you answered your own question. It would be interesting to see the technology develop through the story, rather than out of the blue, but the Humans have drones and they aren't even as advanced as the Hai, so I don't think its necessary.